src/applications/deploying-addons.md

```diff
@@ -0,0 +1,101 @@
+# Deploying Add-ons
+
+The goal of `deis addons` is to give users the power to provision consumable services such as
+a postgres database, a minio bucket, or a logstash connection) to their applications. `deis addons`
+can work with both common on-premise services such as a local mysql database or a Redis server
+for a "private" or on-premise service registry, or with public SaaS applications such as MongoLab
+or Papertrail for a public service offering.
+
+`deis addons` is backed by a project called [service-catalog][]. Service-catalog brings integration
+with [service brokers][] to the Kubernetes ecosystem via the [Open Service Broker API][].
+
+Users of Workflow use `deis addons` to provision an add-on offered by [service brokers][] for
+their applications. The end-goal is to provide a way for users to consume services from brokers and
+have their applications use those services without needing detailed knowledge about how those
+services are created or managed.
+
+As an example, most applications deployed by Workflow need a data store of some kind. `deis addons`
+allows applications to consume services like databases that exist somewhere via common environment
+variables like `DATABASE_URL`.
+
+## Listing Available Add-ons
+
+A user can use `deis addons:installed` to see if they have a database provisioned and what plan it
+is using.
+
+```
+$ deis addons:installed | grep -i postgresql
```

Comment:
So basically if an admin creates a new broker resource, it will populate new service classes into the mix for users to query via `deis addons:catalog`

I haven't added addons:catalog into the docs yet but I am going to amend these once again so users can list instances, bindings and service classes.